powermon/commands/reading_definition.py

```diff
@@ -272,7 +309,9 @@ def __init__(
         state_class: str = None,
         icon: str = None
     ):
-        super().__init__(index, name, response_type, "Wh", description, device_class, state_class, icon)
+        super().__init__(index, name, response_type, description, device_class, state_class, icon, unit="Wh")
+        if response_type not in [ResponseType.INT]:
```

Comment:
In the constructor of each Reading type it will check if the provided ResponseType is supported. What type of error should I raise when this happens?

Reply:
i think we should use custom exceptions more - to make future handling easier

Reply:
maybe a match case construct (in the translate_from_raw function), with `case _:` covering the error state and each other state doing the translation 

InvalidResponse or similar error, so we can propagate the error up and deal without crashing